tests/docker/globus.el9.Dockerfile

```diff
@@ -0,0 +1,87 @@
+FROM rockylinux:9
+
+RUN \
+  yum update -y && \
+  yum install -y \
+    pam-devel \
+    python3-jsonschema \
+    epel-release \
+    gcc-c++ \
+    gnupg \
+    make \
+    python3 \
+    python3-pip \
+    rsyslog \
+    sudo \
+    wget \
+    which \
+    diffutils \
+    procps \
+    rpm-build \
+  && \
+  yum clean all && \
+  rm -rf /var/cache/yum /tmp/*
+
+RUN dnf -y install jansson
+RUN rpm --force -i https://dl.rockylinux.org/pub/rocky/9/CRB/x86_64/os/Packages/j/jansson-devel-2.14-1.el9.x86_64.rpm
+RUN dnf -y --enablerepo=crb install libtool-ltdl-devel
+
+# python 2 and 3 must be installed separately because yum will ignore/discard python2
+RUN \
+  yum check-update -q >/dev/null || { [ "$?" -eq 100 ] && yum update -y; } && \
+  yum install -y \
+    python3 \
+    python3-devel \
+    python3-pip \
+  && \
+  yum clean all && \
+  rm -rf /var/cache/yum /tmp/*
+
+RUN python3 -m pip install xmlrunner distro psutil pyodbc jsonschema requests
```

Comment:
Pass `--no-cache-dir` to pip. This applies to all usages of `pip` and `pip3` in Dockerfiles.